ertviz/models/plot_model.py

```diff
@@ -146,10 +155,9 @@ def repr(self):
                 ]
             )
 
-        bin_count = int(math.ceil(math.sqrt(len(data[0]))))
         _max = max(map(max, data))
         _min = min(map(min, data))
-        bin_size = float((_max - _min) / bin_count)
+        bin_size = float((_max - _min) / self.bin_count)
```

Comment:
self._bin_count i think it is called?

Reply:
`self.bin_count` is a property, which recomputes `bin_count` if necessary

Reply:
yes - of course... my mistake 😬